feature/manager.go

```diff
@@ -188,6 +192,12 @@ func newManager(cfg Config, desc setDesc) (*Manager, error) {
 			raw.Unset(lnwire.RouteBlindingOptional)
 			raw.Unset(lnwire.RouteBlindingRequired)
 		}
+
+		if cfg.NoExperimentalEndorsement {
+			raw.Unset(lnwire.ExperimentalEndorsementOptional)
+			raw.Unset(lnwire.ExperimentalEndorsementRequired)
```

Comment:
See BOLT 7, "an origin node MUST NOT set both the optional and mandatory bits". Independently of `NoExperimentalEndorsement` true / false, I believe required should be always `Unset`.